scripts/release/cli.js

```diff
@@ -136,13 +136,15 @@ const { argv } = require('yargs');
         updates.map(pkg => packageVersionUpdate(pkg, releaseType))
       );
       versions = await prompt(versionUpdates);
-
       /**
        * Remove the skipped packages
        */
       Object.keys(versions).forEach(name => {
         if (versions[name] === 'skip') {
           delete versions[name];
+
+          // remove skipped package from updates array
+          updates = updates.filter(pkgName => pkgName !== name);
```

Comment:
`updates` gets overwritten for each skipped package. Is that what you want?

Reply:
yes, it's fine to overwrite it. It's not super efficient, but we don't need to be and it's much less/readable code.